mobile_files/package.json.orig

```diff
@@ -55,7 +55,7 @@
     "react-native-safe-area-view": "0.9.0",
     "react-native-securerandom": "https://github.com/status-im/react-native-securerandom#0.1.1-1",
     "react-native-splash-screen": "3.1.1",
-    "react-native-status-keycard": "https://github.com/status-im/react-native-status-keycard",
+    "react-native-status-keycard": "https://github.com/status-im/react-native-status-keycard#0dfa0ecee6a025d7e727165b16cd364e410950d7",
```

Comment:
Would be nice to have some release :)

Reply:
@jeluard ok, changed to v2.3.1 release version